avocado/plugins/multiplex.py

```diff
@@ -59,6 +59,9 @@ def configure(self, parser):
         parser.add_argument('--list-variant-id', action='store_true',
                             default=False, help="Append variant id to list")
 
+        parser.add_argument('--get-count', action='store_true',
```

Comment:
I'll use just `--count`, but that's a minor issue (naming things is often controversial).